pom.xml

```diff
@@ -29,7 +29,7 @@
 
   <properties>
     <project.jdk7.home>${env.JAVA_HOME}</project.jdk7.home>
-    <project.build.targetJdk>1.7</project.build.targetJdk>
+    <project.build.targetJdk>1.8</project.build.targetJdk>
```

Comment:
Some of our more widely-used opensource projects are also still on Java 7 and I'm not sure that we want to upgrade all of them beyond what downstream consumers may be using.